libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -22,6 +22,27 @@
             "description": "Value that will be passed back to the parent dialog.",
             "default": "dialog.result"
         },
+        "dialogs": {
+            "type": "array",
+            "title": "Dialogs added to DialogSet.",
+            "dialog": {
+                "oneOf": [
+                    {
+                        "$kind": "Microsoft.IDialog",
+                        "pattern": "^(?!(=)).*",
```

Comment:
Do you need this?  You can verify by removing and seeing if the various forms validate against the schema with/without this in VS Code.

Reply:
removed